spec/indexer_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require File.expand_path("#{File.dirname(__FILE__)}/../config/boot")
+
+require 'json'
+require 'tmpdir'
+require 'spec_helper'
+
+describe Indexer do
+  before(:each) do
+    @temp_dir = Dir.mktmpdir
```

Comment:
I initially tried to use `let` here instead of instance variables, but I seem to remember running into a problem trying to use them inside of `before` and `each`?

Reply:
gosh, I think you _should_ be able to use let for your vars, and call `FileUtils` methods on them.